### PR TITLE
Register and tag PkgDev

### PR DESCRIPTION
**Pull request by jakebolewski**

### CODE REVIEW

**Comment by tkelman** [comment]
dup of #3682

File: .test/METADATA.jl [hunk 1]
```diff
@@ -47,37 +47,57 @@ for pkg in readdir("METADATA")
...
+    host = m.captures[4]
+    if host == nothing
+        error("Invalid url $url for package $pkg. Cannot extract host")
+    end
+    path = m.captures[5]
+    if path == nothing
+        error("Invalid url $url for package $pkg. Cannot extract path")
+    end
+    scheme = m.captures[2]
+    if !(ismatch(r"git", scheme) || ismatch(r"https", scheme))
```
Comment (stevengj) [inline .test/METADATA.jl]: I think this should be a separate PR.   We made a conscious decision at some point to standardize on `git://` URLs, so changing that shouldn't be a side-effect of registering a package.

Comment (jakebolewski) [inline .test/METADATA.jl]: I pushed to the wrong branch.
